### PR TITLE
Add note explaining Razor Pages property binding on GET

### PR DESCRIPTION
**Pull request by scottaddie**
Fixes https://github.com/aspnet/Docs/issues/5657

### CODE REVIEW

**Comment by guardrex** [comment]
@Rick-Anderson previously shot this one down. https://github.com/aspnet/Docs/issues/4655 :gun: lol

**Comment by scottaddie** [comment]
@guardrex In conversations with both @rynowak and @isaac2004 on Friday, it was recommended that we add this. In the past, I've also seen customers ask Damian how to do this. There's a need for it. If not in this doc, we should put it somewhere else.

**Comment by guardrex** [comment]
Preach'in to the choir my friend. I opened #4655 because I wanted to do it.

**Comment by scottaddie** [comment]
I can see both sides too, but it's a problem if customers are going to a PM for the answer.

**Comment by isaac2004** [comment]
Or bothering people on the docs team :)

**Comment by rynowak** [comment]
This absolutely needs to be part of the docs for the cases where folks want to need to use it. This is not a bad practice or something we tell users to avoid.

**Comment by Rick-Anderson** [comment]
@rynowak  can we add something like:
Binding on `Get` is generally not required.

It would be nice to provide some background on why Get is excluded by default. Are there case where blindly adding binding on get is problematic? There must be a good reason it was excluded.

For example, you might need to add binding to `Get` when ...

**Comment by Rick-Anderson** [comment]
@scottaddie  play the video: [ `@DamianEdwards explains risk`](https://youtu.be/p7iHB9V-KVU?t=3251) and add those caveats.

**Comment by scottaddie** [comment]
@rynowak @Rick-Anderson Please take another look.

File: aspnetcore/mvc/razor-pages/index.md [hunk 1]
```diff
@@ -148,6 +148,12 @@ The `Customer` property uses `[BindProperty]` attribute to opt in to model bindi
 
 Razor Pages, by default, bind properties only with non-GET verbs. Binding to properties can reduce the amount of code you have to write. Binding reduces code by using the same property to render form fields (`<input asp-for="Customer.Name" />`) and accept the input.
 
+> [!NOTE]
+> For security reasons, you must opt in to binding GET request data to page model properties. Proceed with caution when trusting user input and mapping it to properties.
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/razor-pages/index.md]: >Proceed with caution

That's not very helpful. How about something along the lines.

Verify user input.

Comment (Rick-Anderson) [reply aspnetcore/mvc/razor-pages/index.md]: Should you add. You might need to bind GET to support searching - or something like that. Gives motivation.

**Pull request merged**